pkg/lib/template/template_loader.go

```diff
@@ -303,6 +303,53 @@ func (loader *Loader) Render(
 		"server_url": func() *url.URL {
 			return loader.ServerURL
 		},
+		"int_range": func(s, e interface{}) []int64 {
+			var start, end int64
+			switch v := s.(type) {
+			case int64:
+				start = v
+			case int:
+				start = int64(v)
+			default:
+				panic("Invalid 'start' range value")
+			}
+
+			switch v := e.(type) {
+			case int64:
+				end = v
+			case int:
+				end = int64(v)
+			default:
+				panic("Invalid 'end' range value")
+			}
+
+			// sv := reflect.ValueOf(s)
+			// ev := reflect.ValueOf(e)
+			// var start, end int64
+			// switch sv.Kind() {
+			// case reflect.Int:
+			// 	start = int64(s.(int))
+			// case reflect.Int64:
+			// 	start = s.(int64)
+			// default:
+			// 	return nil
+			// }
+			// switch ev.Kind() {
+			// case reflect.Int:
+			// 	end = int64(e.(int))
+			// case reflect.Int64:
+			// 	end = e.(int64)
+			// default:
+			// 	return nil
+			// }
+			n := end - start
+			result := make([]int64, n)
+			var i int64
```

Comment:
I dont like the `make` if we are not setting `len` or `cap`.

Use 

```go
result := []int64{}
```


instead.